src/main/target/NUCLEOH743/target.h

```diff
@@ -236,6 +236,15 @@
 
 #define USE_ADC
 
+#if 1
```

Comment:
If this if final then the conditional should be removed.

Reply:
Doh!

Reply:
Done.